components/BodyView.js

```diff
@@ -6,25 +6,29 @@ import CaseTable from "./CaseTable.js";
 
 // mui
 import { makeStyles } from "@material-ui/core/styles";
+import Paper from "@material-ui/core/Paper";
+import Container from "@material-ui/core/Container";
 import Grid from "@material-ui/core/Grid";
 
 const useStyles = makeStyles(theme => ({
   body: {
-    marginTop: theme.spacing(12),
-    height: '100%'
+    marginTop: theme.spacing(5),
+    height: '100%',
+    overflow: 'auto',
+    position: 'relative'
   }
 }));
 
 function BodyView(props) {
   const classes = useStyles();
   return (
     <Fragment>
-    <div className={classes.body}>
-      <Grid>
+    <Container className={classes.body}>
+      <Grid direction='column'>
```

Comment:
So I am not 100% sure if this is the proper solution, I was able to make this error go away:

```
Warning: Failed prop type: The prop `direction` of `Grid` must be used on `container`.
    in WithStyles(ForwardRef(Grid)) (at BodyView.js:27)
    in BodyView (at pages/index.js:60)
    in div
    in Styled(MuiBox) (at pages/index.js:59)
    in div
    in ForwardRef(Container)
    in WithStyles(ForwardRef(Container)) (at pages/index.js:58)
    in Context.Provider (at casecontroller.js:89)
    in InitializedProvider (at pages/index.js:55)
    in div (at pages/index.js:54)
    in Context.Provider
    in ThemeProvider (at pages/index.js:53)
    in Fragment (at pages/index.js:51)
    in Home
    in App
    in Context.Provider
    in Context.Provider
    in Context.Provider
    in Context.Provider
```

By trying either this:

```jsx
    <Container className={classes.body}>
      <Grid container direction='column'>
```
or this:


```jsx
    <Container direction='column' className={classes.body}>
      <Grid>
```
So I am not sure what this changes or which one is correct. I kind of guessed on the latter based on my limited understanding of React.

Reply:
Maybe I misunderstand, but it seems the former (`<Grid container direction='column'>`) idiomatic given the demo in the docs below.

https://material-ui.com/fr/components/grid/

Reply:
Did you have a chance to try with the change `<Grid container direction='column'>`? This fixed it for those of us who tested, not sure the severity of the console error.